src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+  const discountThreeDays = 20;
+  const discountSevenDays = 50;
+  const lessThanThreeDaysCost = dailyCost * days;
+  const lessThanSevenDayCost = lessThanThreeDaysCost - discountThreeDays;
+  const moreThanSevenDayCost = lessThanThreeDaysCost - discountSevenDays;
```

Comment:
So I've meant that you can move only the calculation that you repeat `dailyCost * days` to a variable